packages/auth/amplify_auth_cognito/example/lib/screens/confirm_user_attribute.dart

```diff
@@ -86,7 +86,7 @@ class _ConfirmUserAttributeScreenState
           children: [
             const SizedBox(height: 12),
             TextFormField(
-              initialValue: widget.userAttributeKey.toString(),
+              initialValue: widget.userAttributeKey.key,
```

Comment:
Note: The change here is out of scope of this PR, and just to make testing easier. I believe `toString()` use to return `key`.  Since `CognitoUserAttributeKey` is `AWSDebuggable`, that is no longer true.

`userAttributeKey.key` feels a little clunky. I think that was why `toString()` returned the key. Thoughts on calling this `name` rather than `key` in vNext?

Reply:
Yep, good catch. 

`name` conflicts with the `CognitoUserAtributeKey.name` which is why I used `key`.

Reply:
Oh right. i think we have had this conversation lol. I'll think about what else it could be.